messages/edge_storage/messages.go

```diff
@@ -1,41 +1,58 @@
 package edge_storage
 
 const (
-	USAGE        = "edge-storage"
-	USAGE_BUCKET = "bucket"
+	USAGE         = "edge-storage"
+	USAGE_BUCKET  = "bucket"
+	USAGE_OBJECTS = "object"
 
-	SHORT_DESCRIPTION               = "Manages Edge Storage buckets and objects directly through the command line"
-	SHORT_DESCRIPTION_CREATE_BUCKET = "Creates a bucket in Edge Storage"
-	SHORT_DESCRIPTION_LIST_BUCKET   = "List the buckets in Edge Storage"
-	SHORT_DESCRIPTION_DELETE_BUCKET = "Deleted the bucket in Edge Storage"
+	SHORT_DESCRIPTION                = "Manages Edge Storage buckets and objects directly through the command line"
+	SHORT_DESCRIPTION_CREATE_BUCKET  = "Creates a bucket in Edge Storage"
+	SHORT_DESCRIPTION_LIST_BUCKET    = "List the buckets in Edge Storage"
+	SHORT_DESCRIPTION_LIST_OBJECT    = "List the objects in Edge Storage"
+	SHORT_DESCRIPTION_DELETE_BUCKET  = "Deleted the bucket in Edge Storage"
+	SHORT_DESCRIPTION_CREATE_OBJECTS = "Creates a objects in Edge Storage"
```

Comment:
I'd go for

```suggestion
	SHORT_DESCRIPTION_CREATE_OBJECTS = "Creates objects in Edge Storage"
```

or

```suggestion
	SHORT_DESCRIPTION_CREATE_OBJECTS = "Creates an object in Edge Storage"
```